terragrunt/aws/cloudfront/cloudfront.tf

```diff
@@ -0,0 +1,55 @@
+resource "aws_cloudfront_origin_access_identity" "origin_access_identity" {
+  comment = "cloudfront origin access identity"
+}
+
+resource "aws_cloudfront_distribution" "distribution" {
+  for_each            = var.s3_bucket_regional_domain_name
+  default_root_object = "index.html"
+  enabled             = true
+  web_acl_id          = aws_wafv2_web_acl.cds_website_waf.arn
+
+  default_cache_behavior {
+    target_origin_id       = each.value
+    viewer_protocol_policy = "redirect-to-https"
+    allowed_methods        = ["DELETE", "GET", "HEAD", "OPTIONS", "PATCH", "POST", "PUT"]
```

Comment:
Will the website have any POST/PUT operations going through CloudFront?  If not, we could limit this down to just `["GET", "HEAD", "OPTIONS"]`.